lib/pages/chat/chat.dart

```diff
@@ -1413,15 +1421,13 @@ class ChatController extends State<Chat>
         onSelectMessage(event);
         break;
       case ChatContextMenuActions.copyMessage:
-        onSelectMessage(event);
-        copySingleEventAction();
+        copySingleEventAction(event: event);
```

Comment:
in here, you can call directly to `event.copy(...)`

Reply:
check again pls